devices/Ft6xx6x/samples/TouchPanelProcessor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using nanoFramework.Runtime.Events;
+
+namespace NFAppCore2App2
+{
+    public delegate void TouchEventHandler(object sender, TouchEventArgs e);
+    
+    public class TouchPanelProcessor

```

Comment:
if all is static then the class should be static

Reply:
Also, looking at the main code, this class is not needed and you can move all the functions directly to the main program. Including the delegate